source/manual/alerts/jenkins-agent-not-connected-to-master.html.md

```diff
@@ -38,6 +38,14 @@ If the master is failing to SSH into the agent with a message like "Host key ver
 
 ```
 $ gds govuk connect ssh -e ci_master
+$ sudo -u jenkins ssh ci-agent-<n>.blue.integration.govuk-internal.digital
```

Comment:
I know this isn't part of your change but the line above should be:

```
$ gds govuk connect ssh -e integration ci_master
```

Could you update it as part of this change?